src/client/components/pages/parts/merge-queue.js

```diff
@@ -98,6 +98,7 @@ class MergeQueue extends React.Component {
 				<ButtonGroup>
 					<Button
 						bsStyle="success"
+						className="margin-bottom-d5"
```

Comment:
I think if we dont assign margin on first element that would be fine ?

Reply:
I don't think so that would be good :)
If we remove margin from the first element then this is how it's gonna look

![Screenshot from 2021-04-05 12-33-19](https://user-images.githubusercontent.com/55311336/113547334-50d6e780-960b-11eb-9bc6-19160646ec10.png)

Reply:
Oh ! I am sorry I meant that for the last button .

Is it necessary to give margin to last button ?